src/nlp.jl

```diff
@@ -218,6 +218,48 @@ function dual(c::NonlinearConstraintRef)
     return nldata.nlconstr_duals[c.index.value]
 end
 
+"""
+    nl_dual_start_value(model::Model)
+
+Return the current value of the MOI attribute [`MOI.NLPBlockDualStart`](@ref).
+"""
+function nl_dual_start_value(model::Model)
+    return MOI.get(model, MOI.NLPBlockDualStart())
+end
+
+"""
+    set_nl_dual_start_value(
+        model::Model,
+        start::Union{Nothing,Vector{Float64}},
+    )
+
+Set the value of the MOI attribute [`MOI.NLPBlockDualStart`](@ref).
+
+The start vector corresponds to the Lagrangian duals of the nonlinear
+constraints, in the order given by [`all_nl_constraints`](@ref).
```

Comment:
This description is a bit confusing. Let's say I added nonlinear constraints. After reading this I might think I have to query `all_nl_constraints` and construct some mapping to my constraints. It would be helpful to be more explicit about how to construct this vector given nonlinear constraint references.

Reply:
> After reading this I might think I have to query all_nl_constraints and construct some mapping to my constraints

You do have to do this. I've added an example to the docstring.